@here/harp-mapview/lib/MapView.ts

```diff
@@ -2250,6 +2250,17 @@ export class MapView extends THREE.EventDispatcher {
         this.clearTileCache();
     }
 
+    /**
+     * Allows the user to block some areas of the screen.
+     * @note To clear, simply pass in an empty array.
+     * @param screenSpaceBoxes Boxes that are in pseudo screen space, i.e. the center of the screen
```

Comment:
We should not expose this pseudo screen space. I don't know why we use it internally but the API should just take normal screen coordinates.

Reply:
Agreed, I will change. I thought to be performant, it would be good, but ease of use trumps in this regard.